lib/ansible/modules/cloud/google/gcp_storage_object.py

```diff
@@ -130,6 +130,10 @@
 import mimetypes
 import hashlib
 import base64
+try:
```

Comment:
@willthames `gcp_storage_object` is autogenerated using [`magic-modules`](https://github.com/GoogleCloudPlatform/magic-modules/), you may need to open PR against this in order to fix it.

Reply:
Ok, I've raised https://github.com/modular-magician/ansible/pull/328 - seems a ludicrously roundabout process